packages/server/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@uncharted/pantera-server-type-generator",
```

Comment:
```suggestion
  "name": "@uncharted/terarium-server-type-generator",
```

Reply:
this is an internal package that we use, we can't rename

Reply:
fixed